plugins/node/opentelemetry-instrumentation-express/README.md

```diff
@@ -75,6 +76,13 @@ Express instrumentation has few options available to choose from. You can set th
 - `middleware`,
 - `request_handler` is the name for anything that's not a router or a middleware.
 
+`spanNameHook` is invoked with the following information:
```

Comment:
"... with an object containing following properties"* or similar. Essentially I'd love to see something that tells me there are two arguments and that the properties of the first one are X and the second one is a string, etc.

Reply:
updated the wording a bit